deploy/kubernetes/elastic-agent-standalone/elastic-agent-standalone-role.yaml

```diff
@@ -33,3 +33,9 @@ rules:
       - "/metrics"
     verbs:
       - get
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    verbs:
+      - '*'
```

Comment:
Please add only the permissions needed. I guess this only needs `create`/`read`/`update`/`delete`?

Reply:
Good point! It seems that we only need `get`, `create`, `update` based on the source code of the client: https://github.com/kubernetes/client-go/blob/master/tools/leaderelection/resourcelock/leaselock.go#L41-L82

Reply:
Interesting, are they automatically removed by some controller when they are released?

Reply:
I don't think so. What I have seen is that they are left there until they are to be used again or someone manually removes them.